client/src/main/java/org/threadly/litesockets/client/http/HTTPClient.java

```diff
@@ -587,5 +595,22 @@ public MergedByteBuffers getBody() {
     public String getBodyAsString() {
       return body.duplicate().getAsString(body.remaining());
     }
+    
+    public Element getBodyAsXml() throws SAXException {
```

Comment:
I dont completely disagree, but to mean its no more special than getBodyAsString().  Its just a convenience method.  If you really dont want it there I can remove it.